packages/core/package.json

```diff
@@ -4,13 +4,18 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p ."
+    "build": "tsc -p .",
+    "test": "UNLOCK_ENV=test vitest run --coverage --environment=node"
```

Comment:
Please also add `ci` like we do for other packages (and make sure it runs the linter as well!)